test/torchaudio_unittest/metrics.py

```diff
@@ -0,0 +1,20 @@
+from parameterized import parameterized
+from torchaudio.metrics import levenshtein_distance
+from torchaudio_unittest import common_utils
+
+
+class TestLevenshteinDistance(common_utils.TorchaudioTestCase):
+    @parameterized.expand(
+        [
+            ["abc", "abc", 0],
+            ["aaa", "aba", 1],
+            ["aba", "aaa", 1],
+            ["aa", "aaa", 1],
+            ["aaa", "aa", 1],
+            ["abc", "bcd", 2],
+            [["hello", "world"], ["hello", "world", "!"], 1],
```

Comment:
What would [["hello", "world"], "world"] do?

Reply:
It would said that ["hello", "world"] is not the same as "world", so the edit distance would be 1 replacement. This is a particular case of [comment](https://github.com/pytorch/audio/pull/927#discussion_r497630708).

Reply:
We could decide to detect such cases and compute the edit distance between the word, and each of the word, and return a list of edit distance instead.

This then lead to the question: what API do we expect a metric to offer, and how would this one align?

Reply:
If we go that path, then we'd treat as special cases:
- `[Sequence[T]], Sequence[T]]]`
- `[Sequence[T], List[Sequence[T]]]`
- `[List[Sequence[T]], Sequence[T]]`
- `[List[Sequence[T]], List[Sequence[T]]]`

The last could be a little ambiguous. batch compare 1-1, compare all possible pairs, or edit distance between the two lists? the first seems most likely. Thoughts?

The two current use cases are 
- two strings
- two sentences (list of strings)

Reply:
I'd stick to "two strings" and add the other behavior later on, if necessary. I know you already use it in your example, but how much worse does the code become?

Reply:
I'm not sure I follow your suggestiong: are you saying the function only handles a reference and hypothesis string: `levenshtein_distance("Hello", "ello")`?

How do you suggest comparing the edit distance between the sentences "Hello World!" and "Bonjour  World!"? Right now, the sentence is first split into by spaces, and the distance function is then applied on the two lists of words: `levenshtein_distance(["Hello", "World!"], ["Bonjour", "World!"])`.